optimizely/project_config.py

```diff
@@ -598,3 +638,52 @@ def get_variation_from_key_by_experiment_id(self, experiment_id, variation_key):
                           variation_key, experiment_id)
 
         return {}
+
+    def get_flag_variation_by_id(self, flag_key, variation_id):
+        """
+        Gets variation by id.
+        variation_id can be a string or in case of forced decisions, it can be an object.
+
+        Args:
+            flag_key: flag key
+            variation_key: variation id
+
+        Returns:
+            Variation as a map.
+        """
+
+        if not flag_key:
+            return None
+
+        variations = self.flag_variations_map.get(flag_key)
+        for variation in variations:
```

Comment:
same code is being used in optimizely.py, can you please consolidate.

Reply:
@msohailhussain  I created one function from them Please have a look in the new commits.